serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/service/krr/KrrService.java

```diff
@@ -13,15 +14,16 @@
 public class KrrService {
 
     private ServiceregistryProperties properties;
+    private KeystoreHelper keystoreHelper;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "keystoreHelper" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)